README.md

```diff
@@ -824,6 +825,228 @@ and Equinox will supply the _initial_ value for the `project` function to render
+      - Equinox in-stream snapshots sometimes help but there are tradeoffs)
+    - `Equinox.CosmosStore`
+      - etag-checked read caching (use without that is not recommended in general, though you absolutely will and should turn it off for some streams)
+      - multiple events are packed into each document (critical to avoid per document overhead - this is configurable)
+      - etag-checked RollingState access mode enables allow you to achieve optimal perf and RU cost via the same API without writing an event every time
+
+**The provision of the changefeed needs to be considered as a primary factor in the overall design if you're trying to build a general store - the nature of what you are seeking to provide (max latency and ordering guarantees etc) will be a major factor in designing the schema for how you manage the encoding and updating of the items in the store**
+
+#### Sagas?
+
+>In the system we would like to block particular bike for the user. But at the same time create a reservation for them to store all important business information. So we use and save data to the 2 different streams of data - typical ES problem. We could use event handers / sagas but it brings another level of complexity. 
+
+There will always be complexity in any interesting system that should not just be a CRUD layer over a relational DB.
+Taking particular solution patterns off the table from the off is definitely something you need to be careful to avoid.
+As an analogy: Having lots of classes in a system can make a mess. But collapsing it all into as few as possible can result in ISP and SRP violations and actually make for a hard to navigate and grok system, despite there being less files and less lines of code (aka complexity). Coupling things can sometimes keep things Simple, but can also sometimes simply couple things.
+In my personal experience
+1) Sagas and PMs can be scary, and there are not many good examples out there in an event sourcing context
+2) You can build a massive number of systems without them
+
+But, also IME:
+3) They're pretty fundamental
+4) They are not as hard as you think when you've done two of them
+5) Proper stores enable good ways to manage them
+6) They enable you to keep the overall complexity of a system down by decoupling things one might artificially couple if you're working with a toolbox where you've denied yourself a space for PMs and Sagas
+
+In other words, my YAGNI detector was on high alert for it, as it seems yours is :wink:
+
+#### Transactional writes?
+
+>In the case above we could assume that data inside a single city will be so small, that even with the long usage it won't complete the whole CosmosDB partition. So we could use it to handle saving 2 events in the same time.
+
+For avoidance of doubt: being able to write two events at the same time is a pretty valid thing to want and need to do (Equinox itself, and any credible Event Store supports it)
```

Comment:
**NOTE:** ESDB only for the particular stream.

Reply:
Clarifying this to make it less ambiguous